src/python/pants/backend/python/framework/stevedore/setup_py_kwargs.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from collections import defaultdict
+from dataclasses import dataclass
+
+from pants.backend.python.framework.stevedore.target_types import (
+    ResolvedStevedoreEntryPoints,
+    ResolveStevedoreEntryPointsRequest,
+    StevedoreEntryPoints,
+    StevedoreEntryPointsField,
+    StevedoreNamespaceField,
+)
+from pants.backend.python.goals.setup_py import SetupKwargsRequest
+from pants.base.specs import DirGlobSpec, RawSpecs
+from pants.engine.addresses import Address
+from pants.engine.rules import Get, MultiGet, collect_rules, rule
+from pants.engine.target import Targets
+from pants.util.frozendict import FrozenDict
+from pants.util.logging import LogLevel
+
+
+@dataclass(frozen=True)
+class StevedoreSetupKwargs:
+    """
+    kwargs = {"entry_points": {"stevedore.extension.namespace": ("entry = p.o.i.n:t")}
```

Comment:
Is the last element meant to be a tuple?
```suggestion
    kwargs = {"entry_points": {"stevedore.extension.namespace": ("entry = p.o.i.n:t",)}
```